[docs][installation] Add FSL Prelude as dependency

jidicula opened the pull request:
## Checklist

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied the relevant labels to this PR
- [ ] I've added relevant tests for my contribution
- [x] I've updated the documentation and/or added correct docstrings
- [x] I've assigned a reviewer

<!--- For the title, please observe the following rules:
	- Provide a concise and self-descriptive title
	- Do not include the applicable issue number in the title, do it in the PR body
	- If the PR is not ready for review, convert it to a draft.
-->

## Description
<!-- describe what the PR is about. Explain the approach and possible drawbacks.It's ok to repeat some text from the related issue. -->
**Why this change was necessary**
FSL Prelude is a critical dependency that shimming-toolbox needs to
run. It was previously not included in the installation instructions
in the README for user installation nor in RTD for development
installation.

**What this change does**
See title

**Any side-effects?**
None

## Linked issues
<!-- If the PR fixes any issues, indicate it here with issue-closing keywords: e.g. Resolves #XX, Fixes #XX, Addresses #XX. Note that if you want multiple issues to be autoclosed on PR merge, you must use the issue-closing verb before each relevant issue: e.g. Resolves #1, Resolves #2 -->
Resolves #122 - FSL and prelude not included in installation documentation

kousu commented:
This branch **cannot** be `gh-pages`. As you probably know

https://help.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites
> The default publishing source for a project site is the gh-pages branch. If the repository for your project site has a gh-pages branch, your site will publish automatically from that branch.

This PR needs to be closed, the branch renamed, and resubmitted.

kousu commented:
I got `mkdocs` running on `tristano`, please go forth and copy: https://github.com/shimming-toolbox/shimming-toolbox.org/pull/31/

po09i commented:
Closing PR to change branch name.

jidicula committed:
[docs][installation] Add FSL Prelude as dependency

**Why this change was necessary**
FSL Prelude is a critical dependency that shimming-toolbox needs to
run. It was previously not included in the installation instructions
in the README for user installation nor in RTD for development
installation.

**What this change does**
See title

**Any side-effects?**
None

**Additional context/notes/links**

Resolves #122 - FSL and prelude not included in installation documentation

jidicula commented:
Possibly related to #95
